kurtosis-devnet/pkg/kurtosis/endpoints.go

```diff
@@ -212,6 +212,43 @@ func (spr serviceParserRules) apply(serviceName string, endpoints descriptors.En
 	return nil
 }
 
+// TODO: this might need some adjustments as we stabilize labels in optimism-package
+const (
+	kindLabel      = "op.kind"
+	networkIDLabel = "op.network.id"
+	nodeIndexLabel = "op.node.idx"
```

Comment:
I used `op.network.participant.name` in `optimism-package` since it's no longer an index but a hashmap key

Reply:
done